doc/source/dev/contributor/benchmarking.rst

```diff
@@ -0,0 +1,208 @@
+.. _benchmarking-with-asv:
+
+Benchmarking SciPy with Airspeed Velocity
+=========================================
```

Comment:
New. Contains most of what `benchmarks\README.rst` (not linked from documentation) has and much more.